### DISCUSSION TITLE
Lession 12: Question About Stablecoin Liquidators

### DISCUSSION THREAD

**Discussion by terranweb3** [Cyfrin/foundry-full-course-cu#619] [Q&A]
Hello guys, I have a question about liquidators in the stablecoin system in lesson 12.

As I understand it, liquidators are anyone who can join to liquidate user mint ETH debt when collateral goes down in price. So in practice, will we have an interface for users (liquidators) to see which users are reaching the liquidation threshold and liquidate to make 10% profit if needed? Or will there be an automated process on the backend to continuously scan and find which users are reaching the liquidation threshold and liquidate them automatically?

Thanks for your replies!

**Comment by theirrationalone** [comment]
Walk through [this](https://github.com/Cyfrin/foundry-full-course-f23/discussions/523#discussioncomment-6630649) 

And nothing is automated anywhere, main, test, fork

**Reply by terranweb3** [reply]
I have read through this library of makerdao in liquidation of over-mortgage loans. It is worth mentioning that I have no knowledge of Python. https://github.com/makerdao/market-maker-keeper

**Reply by theirrationalone** [reply]
Btw, that is now archived by them.... But you can acquire the knowledge of python for future.
